akka-http-core/src/main/scala/akka/http/impl/util/StreamUtils.scala

```diff
@@ -275,9 +286,26 @@ private[http] object StreamUtils {
 /**
  * INTERNAL API
  */
-private[http] class EnhancedByteStringSource[Mat](val byteStringStream: Source[ByteString, Mat]) extends AnyVal {
+@InternalApi private[http] class EnhancedByteStringSource[Mat](val byteStringStream: Source[ByteString, Mat]) extends AnyVal {
```

Comment:
Let's just make these `private`. No need to add those annotations everywhere if the owning object has it already.